documentation.md

```diff
@@ -0,0 +1,155 @@
+#  ScamGuard Command Documentation
```

Comment:
I'm wondering if we want to put this into the wiki first, I forgot to actually turn on the wiki setting on github.

Reply:
verbiage changed.